### DISCUSSION TITLE
Dashboard

### DISCUSSION THREAD

**Discussion by DonCanDo** [dewenni/ESP_Buderus_KM271#136] [Ideas]
Hey,

danke dass du dich immer für kümmerst. Kannst du beim nächsten Update auf dem Dashboard die Warmspeicherpumpe und die Zirkulationspumpe hinzufügen und die Raum Ist Soll Temperatur?

Danköööö

**Comment by dewenni** [comment]
Raum Ist / Soll Temperatur würde ich ungern auf das Dashboard machen weil ein vermutlich bei einigen (und bei mir) keine sinnvollen Werte liefert. Ich habe keinen Raumfühler.

Die Zirkulationspumpe für den Heizkreis habe ich ja schon drin.
Bei der Warmwasser Zirkulationspumpe ist das wieder so eine Sache. Die hat nicht jeder (ich z.B. habe auch keine).

Wenn ich also anfange und hier Werte darstelle die nicht jeder hat, dann müsste ich das wieder konfigurierter machen.
Den Aufwand möchte ich aber im Moment zumindest nicht treiben.

**Comment by DonCanDo** [comment]
Alles weiß ich bescheid. 

Danke trotzdem =)
